src/wallet/wallet.cpp

```diff
@@ -433,6 +433,8 @@ void CWallet::Flush(bool shutdown)
     database->Flush(shutdown);
 }
 
+std::atomic<bool> CWallet::m_end_flush{false};
```

Comment:
Why not use default member initialization in the header file?

Reply:
This wouldn't compile. Thanks C++